ansible/library/extract_log.py

```diff
@@ -215,6 +215,7 @@ def combine_logs_and_save(directory, filenames, start_string, target_filename):
                         do_copy = True
                     if do_copy:
                         fp.write(line)
+        fp.flush()
```

Comment:
Why do we need to do flush here? ```with open(target_filename, 'w') as fp:``` closes the file at the end of with block which means it will flush before close